spec/models/form_handler_spec.rb

```diff
@@ -34,6 +34,23 @@
       expect(all_forms.count).to be >= 1
       expect(all_forms["current_sales"]).to be_a(Form)
     end
+
+    context "when in 23/24 period or later" do
+      let(:now) { Time.utc(2023, 6, 7) }
+
+      around do |example|
+        Timecop.freeze(now) do
+          Singleton.__init__(described_class)
+          example.run
+        end
+        Singleton.__init__(described_class)
```

Comment:
I don't think we need this line because it's added in spec_helper after hook

Reply:
excellent ta. removed it somewhere else too